### PR TITLE
Put double quotes in error messages with resources

### PR DESCRIPTION
**Pull request by etefera**
Fixes https://github.com/GoogleContainerTools/kpt/issues/964

### CODE REVIEW

File: internal/cmdexport/pathutil/pathutil.go [hunk 1]
```diff
@@ -55,14 +55,14 @@ func Exists(path string) bool {
 func IsInsideDir(path string, directory string) (isInside bool, err error) {
 	if !filepath.IsAbs(path) {
 		err = fmt.Errorf(
-			"argument `path` (%s) is not an absolute path",
+			"argument `path` ('%s') is not an absolute path",
```
Comment (mikebz) [inline internal/cmdexport/pathutil/pathutil.go:58]: what about https://www.geeksforgeeks.org/how-to-print-string-with-double-quotes-in-golang/

Comment (etefera) [reply internal/cmdexport/pathutil/pathutil.go:58]: Responding to the title of https://github.com/GoogleContainerTools/kpt/issues/964
Is there a strong preference for single vs. double quotes? @phanimarupaka

Comment (phanimarupaka) [reply internal/cmdexport/pathutil/pathutil.go:58]: @mikebz comment makes more sense in this context. We already have some examples in the codebase. https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/cmdset.go#L131

File: internal/cmdexport/pathutil/pathutil_test.go [hunk 2]
```diff
@@ -80,9 +80,9 @@ func TestIsInsideDirectory(t *testing.T) {
 	)
 
 	_, err := IsInsideDir("resources", ".")
-	assert.Error(t, err, "argument `path` (resources) is not an absolute path")
+	assert.Error(t, err, "argument `path` ('resources') is not an absolute path")
```
Comment (phanimarupaka) [inline internal/cmdexport/pathutil/pathutil_test.go:83]: Do we need quotes around resources here ?

File: internal/cmdexport/pathutil/pathutil_test.go [hunk 3]
```diff
@@ -80,9 +80,9 @@ func TestIsInsideDirectory(t *testing.T) {
...
 	_, err = IsInsideDir("/resources", ".")
-	assert.Error(t, err, "argument `directory` (.) is not an absolute path")
+	assert.Error(t, err, "argument `directory` ('.') is not an absolute path")
```
Comment (phanimarupaka) [inline internal/cmdexport/pathutil/pathutil_test.go:85]: Do we need quotes here ?

**Comment by phanimarupaka** [comment]
Please use double quotes. %q is good way to go instead of %s. Does %q work fine instead of %v ?

File: internal/cmdexport/pathutil/pathutil_test.go (hunk 2 above)
Comment (etefera) [reply internal/cmdexport/pathutil/pathutil_test.go]: Removed.

File: internal/cmdexport/pathutil/pathutil_test.go (hunk 3 above)
Comment (etefera) [reply internal/cmdexport/pathutil/pathutil_test.go]: Removed.

File: internal/cmdexport/pathutil/pathutil.go (hunk 1 above)
Comment (etefera) [reply internal/cmdexport/pathutil/pathutil.go]: Done.

File: pkg/live/inventoryrg.go [hunk 4]
```diff
@@ -133,7 +133,7 @@ func (icm *InventoryResourceGroup) Load() ([]object.ObjMetadata, error) {
 			Group: strings.TrimSpace(group),
 			Kind:  strings.TrimSpace(kind),
 		}
-		klog.V(4).Infof("creating obj metadata: %s/%s/%s", namespace, name, groupKind)
+		klog.V(4).Infof("creating obj metadata: \"%s/%s/%s\"", namespace, name, groupKind)
```
Comment (phanimarupaka) [inline pkg/live/inventoryrg.go:136]: You can use `creating obj metadata: `"%s/%s/%s"`to avoid escape

File: pkg/live/inventoryrg.go [hunk 5]
```diff
@@ -162,7 +162,7 @@ func (icm *InventoryResourceGroup) GetObject() (*unstructured.Unstructured, erro
 	klog.V(4).Infof("Creating list of %d resources", len(icm.objMetas))
 	var objs []interface{}
 	for _, objMeta := range icm.objMetas {
-		klog.V(4).Infof("storing inventory obj refercence: %s/%s", objMeta.Namespace, objMeta.Name)
+		klog.V(4).Infof("storing inventory obj reference: \"%s/%s\"", objMeta.Namespace, objMeta.Name)
```
Comment (phanimarupaka) [inline pkg/live/inventoryrg.go:165]: Same as above

File: pkg/live/rgpath.go [hunk 6]
```diff
@@ -41,7 +41,7 @@ func (p *ResourceGroupPathManifestReader) Read() ([]*unstructured.Unstructured,
 	inv := kf.Inventory
 	invObj, err := generateInventoryObj(inv)
 	if err == nil {
-		klog.V(4).Infof("from Kptfile generating ResourceGroup inventory object %s/%s/%s",
+		klog.V(4).Infof("from Kptfile generating ResourceGroup inventory object \"%s/%s/%s\"",
```
Comment (phanimarupaka) [inline pkg/live/rgpath.go:44]: Same as above

File: pkg/live/rgstream.go [hunk 7]
```diff
@@ -89,9 +89,9 @@ func transformKptfile(resource []byte) (*unstructured.Unstructured, error) {
 		return nil, err
 	}
 	if kptFileTemplate.ResourceMeta.TypeMeta != kptfile.TypeMeta.TypeMeta {
-		return nil, fmt.Errorf("invalid kptfile type: %s", kptFileTemplate.ResourceMeta.TypeMeta)
+		return nil, fmt.Errorf("invalid kptfile type: %q", kptFileTemplate.ResourceMeta.TypeMeta)
 	}
 	inv := kptFileTemplate.Inventory
-	klog.V(4).Infof("generating ResourceGroup inventory object %s/%s/%s", inv.Namespace, inv.Name, inv.InventoryID)
+	klog.V(4).Infof("generating ResourceGroup inventory object \"%s/%s/%s\"", inv.Namespace, inv.Name, inv.InventoryID)
```
Comment (phanimarupaka) [inline pkg/live/rgstream.go:95]: Same as above

**Review by phanimarupaka — APPROVED** [review]

**Pull request merged**